tutorials/202103/startup.m

```diff
@@ -0,0 +1,7 @@
+
+addpath(genpath('../../../U19-pipeline-matlab'))
```

Comment:
Unsure of the need to commit this as well. It would need to manually copied over into your startup directory anyway. Why not just make it a script that your docs indicate should be ran first to set things up?